src/translations/en/notFound.json

```
@@ -0,0 +1,4 @@
+{
+  "turn_back" : "Turn back to save the planet !",
```

```suggestion
  "turn_back" : "Go back save the planet !",
```
C'est mieux je pense ça